lib/src/native/realm_core.dart

```diff
@@ -1078,6 +1078,22 @@ class _RealmCore {
       return userHandles;
     });
   }
+
+  void switchUser(App application, User user) {
+    return using((arena) {
+      final newUserPtr = arena<Pointer<realm_user>>();
+      _realmLib.invokeGetBool(
+          () => _realmLib.realm_app_switch_user(
+                application.handle._pointer,
+                user.handle._pointer,
+                newUserPtr,
```

Comment:
You can pass `nullptr` here.

Reply:
good catch. I was staring at this code and still missed it